packages/bpk-component-skip-link/src/BpkSkipLink-test.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import BpkSkipLink from './BpkSkipLink';
+
+describe('BpkSkipLink', () => {
+  it('should render correctly', () => {
+    const tree = renderer.create(<BpkSkipLink />).toJSON();
```

Comment:
The docs say href and label are required as props yet we are not passing them here, is that expected?